tests/foreman/ui/test_ansible.py

```diff
@@ -742,9 +744,37 @@ def test_positive_install_ansible_collection_via_job_invocation(self):
             6. Click "Submit"
 
         :expectedresults: The Ansible collection is successfully installed on the host
-
-        :CaseAutomation: NotAutomated
         """
+        client = rhel_contenthost
+        # Enable Ansible repository and Install ansible or ansible-core package
+        client.register(module_org, None, module_ak_with_cv.name, target_sat)
+        client.create_custom_repos(rhel8_aps=settings.repos.rhel8_os.appstream)
```

Comment:
Please update it to fetch the repos based on the rhel version.

Reply:
Done